tests/test_safety_gaps.py

```
@@ -0,0 +1,74 @@
+from tests.conftest import run
+
+
+class TestTopLevelReturn:
+    def test_return_outside_function_is_structure_error(self):
+        with pytest.raises(TarteebJeGhalti, match="wapas"):
+            run("wapas 5")
+
+
+class TestDuplicateKeyword:
+    def test_duplicate_keyword_argument_raises_arity_error(self):
+        with pytest.raises(MatalabJeGhalti, match="(?i)dobara"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)